packages/xrpl/src/models/common/index.ts

```diff
@@ -148,3 +148,10 @@ export interface AuthAccount {
     Account: string
   }
 }
+
+export interface XChainBridge {
```

Comment:
Is this an array? (I get a little confused reading this type on the way you're supposed to reference it since `XChainBridge.LockingChainDoor` doesn't seem right).

Reply:
It's an `object` - the type's name is `XChainBridge`, so it would be `varName: XChainBridge = {...}; varName.LockingChainDoor`